script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -519,27 +545,31 @@ But what should we do?
 [sha:24ecb1490fdaafb68862b5b172004917ba091f2f]{
 -- Page 409, Offset 9667.
 -- わたしはこのまま始めても構わないけど」
-I don't mind starting with things the way they are, but..."
+// Man this whole exchange is just not great.
+// Doesn't help that it feels like kinda a leap in JP too?
+I'm fine with this place as the battlefield if you are, but..."
```

Comment:
maybe something like
>I'm ready right now if you are, just so you know.

Reply:
But then how does this link to 9669? That's why I've crammed 'battlefield' in here. Otherwise it's just a totally bonkers jump.

Reply:
Well, the original is a bonkers jump...
Feels like hand-holding to me.